gateway/xgress_edge_wss/listener.go

```diff
@@ -0,0 +1,450 @@
+/*
```

Comment:
I feel like the vast majority of this code could be shared between implementations (I could be wrong, I haven't tried it). I'd be surprised if we couldn't pull at least some of this into a shared package, though

Reply:
I agree.  My initial goal was to get things flowing end-to-end (after being away from it for so long), using separate "wss" code, staying away from existing xgress_edge mechanisms.  We can now look into refactoring all xgress_edge_* code using shared code where possible.

Reply:
deleted.  new wss transport is in place.